src/invocation_settings.rs

```diff
@@ -0,0 +1,35 @@
+use std::{fs, path::Path};
+
+use anyhow::{Context, Result as AnyResult};
+use semver::Version;
+use serde::{Deserialize, Serialize};
+
+/// The invocation settings.
+///
+/// When cargo-breaking is invoked by the user via CLI, cargo-breaking must
+/// generate a bunch of things. For some reasons, some must be randomely
```

Comment:
```suggestion
/// generate a bunch of things. For some reasons, some must be randomly
```

"In order to make sure we won't clash with an already existing crate name when we call rustc,..."